src/deepqmc/wf/baseline/pyscfext.py

```diff
@@ -5,12 +5,10 @@
 from pyscf.mcscf import CASSCF
 from pyscf.scf import RHF
 
-from ...utils import pad_list_of_3D_arrays_to_one_array
-
 log = logging.getLogger(__name__)
 
 
-def pyscf_from_mol(mol, basis, cas=None, **kwargs):
+def pyscf_from_mol(hamil, basis, cas=None, **kwargs):
```

Comment:
We should update the name and docstring of this function `mol`->`hamil`.

Reply:
Good point, I've updated the docstring